test/tests_for_integration/api_to_db_test.py

```diff
@@ -146,6 +178,11 @@ def service(full_config, auth_url, mongo_client, config):
 
     # from this point on, calling the get_*_test_config methods will get the temp config file
     os.environ[KB_DEPLOY_ENV] = cfgpath
+    # The server creates the configuration, impl, and application *AT IMPORT TIME* so we have to
+    # import *after* setting the config path.
+    # This is terrible design. Awful. It definitely wasn't me that wrote it over Xmas in 2012
```

Comment:
😀  😢 

Do you need to add your name to that comment so people don't have to use git blame to work out who wrote it?

Reply:
The shame is too great, I must hide my face from the world.